habitat/sims/pyrobot/pyrobot.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Any
+
+import numpy as np
+from gym import Space, spaces
+
+import pyrobot
+from habitat.core.registry import registry
+from habitat.core.simulator import (
+    BumpSensor,
+    Config,
+    DepthSensor,
+    RGBSensor,
+    SensorSuite,
+    Simulator,
+)
+from habitat.core.utils import try_cv2_import
+
+cv2 = try_cv2_import()
+
+
+def _locobot_base_action_space():
+    return spaces.Dict(
+        {
+            "go_to_relative": spaces.Box(low=-np.inf, high=np.inf, shape=(3,)),
+            "go_to_absolute": spaces.Box(low=-np.inf, high=np.inf, shape=(3,)),
+        }
+    )
+
+
+def _locobot_camera_action_space():
+    return spaces.Dict(
+        {
+            "set_pan": spaces.Box(low=-np.inf, high=np.inf, shape=(1,)),
```

Comment:
What is `pan/tilt` unit of measurements? If that's degrees then we can specify proper limits.

Reply:
The PAN and TILT from pyrobot aren't restricted to an angle range like `(-pi, pi)`.